core/sql/optimizer/SynthType.cpp

```diff
@@ -6699,16 +6699,20 @@ const NAType *LOBoper::synthesizeType()
 const NAType *LOBinsert::synthesizeType()
 {
   // Return blob type
-
-  ValueId vid1 = child(0)->getValueId();
-  const NAType &typ1 = (NAType&)vid1.getType();
+  ValueId vid1;
+  const NAType *typ1 = NULL;
+  if (child(0))
+    {
+      vid1 = child(0)->getValueId();
+      typ1 = &vid1.getType();
+    }
 
   if ((obj_ == STRING_) ||
       (obj_ == FILE_) ||
       (obj_ == EXTERNAL_) ||
       (obj_ == LOAD_))
     {
-      if (typ1.getTypeQualifier() != NA_CHARACTER_TYPE)
+      if (typ1->getTypeQualifier() != NA_CHARACTER_TYPE)
```

Comment:
Could typ1 be null here? (Looks like it will be if child(0) is null.) Should we check for this?